node.go

```diff
@@ -248,6 +248,37 @@ func ReconfigureMembershipExt(dir string, cluster []NodeInfo) error {
 	return server.RecoverExt(cluster)
 }
 
+// LastEntryInfo holds information about the last entry in the persistent raft
+// log of a node.
+//
+// The order of fields is significant and ensures that the comparison `x > y`
+// of two LastEntryInfo values is equivalent to asking whether the log of `x`
+// is more up to date than the log of `y`.
+type LastEntryInfo struct {
+	Term, Index uint64
+}
+
+// ReadLastEntryInfo reads information about the last entry in the raft
+// persistent log from a node's data directory.
+//
+// This is a non-destructive operation, but is not read-only, since it has the
+// side effect of renaming raft open segment files to closed segment files.
+func ReadLastEntryInfo(dir string) (LastEntryInfo, error) {
+	server, err := bindings.NewNode(context.Background(), 1, "1", dir)
```

Comment:
I guess this is creating a new with a fake number and address, hence the 1 and "1" is not important. My question is, can this id clash with some existing node?

Reply:
Indeed the ID and address are fake, and they can alias with the ID/address of a node that's present in the cluster configuration. However, it doesn't matter because dqlite_node_create doesn't take any action based on the provided ID and address, it just stores them away for dqlite_node_start to use later.